vms/platformvm/txs/executor/camino_tx_executor.go

```diff
@@ -2364,3 +2365,25 @@ func outputsAreEqual(outs1, outs2 []*avax.TransferableOutput) bool {
 		return ok && out1.Asset == out2.Asset && outEq1.Equal(out2.Out)
 	})
 }
+
+func getBaseFee(s state.Chain, cfg *config.Config) (uint64, error) {
+	fee, err := s.GetBaseFee()
+	switch err {
+	case database.ErrNotFound:
+		return cfg.TxFee, nil
+	case nil:
+		return fee, nil
+	}
+	return 0, err
+}
+
+func getFeeDistribution(s state.Chain, cfg *config.Config) ([dacProposals.FeeDistributionFractionsCount]uint64, error) { //nolint:unused
```

Comment:
where's this func used?

Reply:
it's not used though anywhere. and you ve added a linter suppress unused

Reply:
Yes. Cause initially, we had only exported State.GetFeeDistribution method, which is allowed to be not-used cause of export. But now, cause part of logic is moved outside of this method, we need to define it somewhere, so I created this method along with `getBaseFee`, but this one is not exported and is not used yet (as was exported `GetFeeDistribution`)

Reply:
maybe add a TODO explaining why we need in the future